.github/labeler.yml

```diff
@@ -0,0 +1,91 @@
+# This configures label matching for PR's.
+#
+# The keys are labels, and the values are lists of minimatch patterns
+# to which those labels apply.
+#
+# NOTE: This can only add labels, not remove them.
+# NOTE: Due to YAML syntax limitations, patterns or labels which start
+# with a character that is part of the standard YAML syntax must be
+# quoted.
+#
+# Please keep the labels sorted and deduplicated.
+
+area/backends:
+  - backends/*
+  - backends/**
+  - exporting/*
+  - exporting/**/*
+
+area/ci:
+  - .travis/*
+  - .travis/**/*
+  - .github/*
+  - .github/**/*
+
+area/collectors:
+  - collectors/*
+  - collectors/**/*
+
+area/daemon:
+  - daemon/*
+  - daemon/**/*
+
+area/database:
+  - database/*
+  - database/**/*
+
+area/docs:
+  - "**/*.md"
+  - diagrams/*
+  - diagrams/**/*
+  - netlify.toml
+
+area/external/python:
+  - collectors/python.d.plugin/*
+  - collectors/python.d.plugin/**/*
+
+area/external:
+  - collectors/charts.d.plugin/*
+  - collectors/charts.d.plugin/**/*
+  - collectors/node.d.plugin/*
+  - collectors/node.d.plugin/**/*
+
+area/health:
+  - health/*
+  - health/**/*
+
```

Comment:
I think we are missing `area/libnetdata` , because sometimes it is necessary to append functions inside it.

Reply:
This PR isn't to change what labels we have, just how they get applied to PR's (there actually is a GitHub Action available for managing what labels a repo has as part of the repo itself, but that's a bit out of scope for this PR IMO).

However, I do think that it would be a good idea to add such a label, as nothing else we have currently covers that part of the code. If someone on the core team wants to add it, I'll be happy to update the PR to include a labeling rule for it.

Reply:
All right, I confess I do not see any motive for we do not have labels for `libnetdata`. It is expected that we will have big changes on it next year when we begin to bring support to Microsoft.